cmd/earthly/flag/global.go

```diff
@@ -539,5 +540,12 @@ func (global *Global) RootFlags(installName string, bkImage string) []cli.Flag {
 			Destination: &global.NoAutoSkip,
 			Value:       false,
 		},
+		&cli.BoolFlag{
```

Comment:
@alexcb 
Should this be a hidden flag?

Reply:
we were thinking some users might want to run `earthly --github-annotations` directly on their terminal (potentially as a way to experiment with the github actions output).

I think it's fine left visible; see https://github.com/earthly/earthly/pull/4047#discussion_r1578134238